docs/data_prep.md

```diff
@@ -10,17 +10,29 @@ please prepare the tabular data for your dataset as a single, tab-separated file
 
 ## General requirements for the phenotypic TSV
 
-- The TSV must contain a minimum of two columns: at least one column must contain subject IDs, 
+### All datasets
+
+A valid dataset for Neurobagel **must** include a TSV file that describes participant attributes. 
+The TSV must contain a minimum of two columns: at least one column must contain subject IDs, 
 and at least one column must describe demographic or other phenotypic information 
 (for variables currently modeled by Neurobagel, see the [data dictionary section](dictionaries.md)).
-- If the dataset has a corresponding BIDS directory (i.e., imaging data), 
-at least one column in the TSV must contain subject IDs that match the names of [BIDS subject subdirectories](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#filesystem-structure). 
-Further, the IDs in this column must be _the same or a superset_ of the subject labels in the BIDS directory. 
-That is, Neurobagel does not currently allow for datasets where subjects have BIDS data but are not represented in the phenotypic TSV.
 
-## Accepted forms of tabular data
+### Datasets with imaging (BIDS) data
+
+If a dataset has imaging data in [BIDS](https://bids-specification.readthedocs.io/en/stable/) format, 
+Neurobagel **additionally** requires that:
+
+- At least one column in the phenotypic TSV contains subject IDs that match the names of [BIDS subject subdirectories](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#filesystem-structure). 
+If this condition is not met, you will encounter an error when [running the Neurobagel CLI](cli.md) on your dataset to generate Neurobagel graph-ready files.
```

Comment:
:cherries: Want to give them a preview / general string of what the error will say? 
Might help a non-technical user who gets freaked out by the massive traceback
but might spot the specific wording.